tool/tctl/common/collection.go

```diff
@@ -126,16 +137,32 @@ func (s *serverCollection) resources() (r []types.Resource) {
 }
 
 func (s *serverCollection) writeText(w io.Writer) error {
-	t := asciitable.MakeTable([]string{"Nodename", "UUID", "Address", "Labels"})
+	var rows [][]string
 	for _, s := range s.servers {
-		t.AddRow([]string{
-			s.GetHostname(), s.GetName(), s.GetAddr(), s.LabelsString(),
+		addr := s.GetPublicAddr()
```

Comment:
I'm not 100% in which case we set the server's public address, but I think it only may make sense for proxies. I think this collection represents SSH nodes so we probably only want to show Addr here.